WPMedia/fusion-news-theme-blocks
# ADX-30 - Safari Gallery Bug

**Be sure to run `npm test`, `npm run lint`, and write detailed test steps before requesting reviewers**

## Description

#### Jira Ticket: [ADX-30](https://arcpublishing.atlassian.net/browse/ADX-30)

Changed image height to eliminate scroll bar in gallery in Safari

## Test Steps

_Add detailed test steps a reviewer must complete to test this PR_

1. Checkout this branch `git checkout ADX-30`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/gallery-block`
3. Examine a page with a gallery block in chrome and Safari and make sure there is no weird overflow scrollbar.


[ADX-30]: https://arcpublishing.atlassian.net/browse/ADX-30?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
